e2e/test-plan.md

```
@@ -0,0 +1,964 @@
+
+### 1. Core Playground
+
+**Seed:** `e2e/seed.spec.ts`
+
+#### 1.1. loads-application-with-default-state
+
+**File:** `e2e/playground/loads-application.spec.ts`
+
+**Steps:**
+
+1. Navigate to http://127.0.0.1:5173/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)